modules/admin/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -0,0 +1,162 @@
+    auth:
+      existingSecret: <cred-secret> # inject credentials secret to Backstage <2>
+  backstage:
+    appConfig:
+      backend:
+        database:
+          connection:  # configure Backstage DB connection parameters
+            host: ${POSTGRES_HOST}
+            port: ${POSTGRES_PORT}
+            user: ${POSTGRES_USER}
+            password: ${POSTGRES_PASSWORD}
+            ssl:
+              rejectUnauthorized: true,
+              ca:
+                $file: /opt/app-root/src/postgres-ca.pem
+              key:
+                $file: /opt/app-root/src/postgres-key.key
+              cert:
+                $file: /opt/app-root/src/postgres-crt.pem
+  extraEnvVarsSecrets:
+    - <cred-secret> # inject credentials secret to Backstage <3>
+  extraEnvVars:
+    - name: BACKEND_SECRET
+      valueFrom:
+        secretKeyRef:
+          key: backend-secret
+          name: '{{ include "janus-idp.backend-secret-name" $ }}'
+  extraVolumeMounts:
+    - mountPath: /opt/app-root/src/dynamic-plugins-root
+      name: dynamic-plugins-root
+    - mountPath: /opt/app-root/src/postgres-crt.pem
+      name: postgres-crt # inject TLS certificate to Backstage cont. <4>
+      subPath: postgres-crt.pem
+    - mountPath: /opt/app-root/src/postgres-ca.pem
+      name: postgres-ca # inject CA certificate to Backstage cont. <5>
+      subPath: postgres-ca.pem
+    - mountPath: /opt/app-root/src/postgres-key.key
+      name: postgres-key # inject TLS private key to Backstage cont. <6>
+      subPath: postgres-key.key              
+  extraVolumes:
+    - ephemeral:
+        volumeClaimTemplate:
+          spec:
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: 1Gi
+      name: dynamic-plugins-root
+    - configMap:
+        defaultMode: 420
+        name: dynamic-plugins
+        optional: true
+      name: dynamic-plugins
+    - name: dynamic-plugins-npmrc
+      secret:
+        defaultMode: 420
+        optional: true
+        secretName: dynamic-plugins-npmrc
+    - name: postgres-crt
+      secret:
+        secretName: <crt-secret> <7>
+        # ...
+----
+<1> Set the value of the `upstream.postgresql.enabled` parameter to `false` to disable creating local PostgreSQL instances. 
+<2> Provide the name of the credential secret.
+<3> Provide the name of the credential secret.
+<4> Optional: Provide the name of the TLS certificate only for a TLS connection.
+<5> Optional: Provide the name of the CA certificate only for a TLS connection.
+<6> Optional: Provide the name of the TLS private key only if your TLS connection requires a private key.
+<7> Provide the name of the certificate secret if you have configured a TLS connection.
+
+. Apply the configuration changes in your Helm configuration file:
++
+[source,terminal]
+----
+helm upgrade -n <your-namespace> <your-release-name> redhat-developer/backstage -f values.yaml
```

Comment:
```suggestion
[source,subs="attributes+"]
----
helm upgrade -n <your-namespace> <your-deploy-name> openshift-helm-charts/redhat-developer-hub -f values.yaml --version {product-chart-version}
```

This is the proper Helm chart for downstream. 
And we have to use the version.
And we should change `<your-release-name>` to `<your_deploy_name>` as this is already used before in the docs, and is better

Reply:
Sorry, I deleted my previous suggestion, since this is better and complete